tests/config_test.cc

```diff
@@ -160,4 +160,13 @@ TEST_F(ConfigTest, ConfigDataGetStringRep) {
     EXPECT_FALSE(impl::ConfigDataGetStringRep(SomeRandomClass()).empty());
 }
 
+TEST_F(ConfigTest, WrongFilePath) {
+    ConfigFile empty_config("");
+    ConfigFile invalid_file_config("nonexistent.config");
```

Comment:
.cfg or  .conf?
Also all of them are generally not a good extra because ext should describe format, not use case